### PR TITLE
Make inline video / picture in picture enabled by default on WebView for iOS

### LINKED ISSUE #4807: HTML5 video not working on MAUI WebView
### Description

If I want to add video in my MAUI Blazor App I have a problem with rendering this video on iOS. On Android works well.
I think the problem is in **BlazorWebView** component.

**UPDATE** This applies to all MAUI webviews, so is not Blazor-specific.

### Steps to Reproduce

1. Create a new blank .NET MAUI Blazor App

2. Add some sample video mp4 file into the wwwroot/videos folder
image

3. Replace content of **Index.razor** page with content below:
```
@page "/"

<h1>Hello, world!</h1>

Welcome to your new app.

<video autoplay muted loop playsinline class="slideContent">
    <source src="videos/TRIMOTERM.mp4" type="video/mp4">
</video>
```

3. Run app on Android. All good.
image

4. Run app on iPhone 12 (iOS 15.2) and nothing is shown where video could be.
image

### Version with bug

Preview 13 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iPhone 12 iOS 15.2, Xcode 13.2.1, VS 17.2.0 Preview 1.0

### Did you find any workaround?

No

### Relevant log output

```shell
Nothing
```

### PR DESCRIPTION
**Pull request by Redth**
### Description of Change

By default, iOS does not allow inline videos.  This changes the defaults to enable them.  
On iOS you can't change these settings after passing the `WKWebViewConfiguration` into the `WKWebView` constructor, so they have to be set initially.

You can still override the WebViewHandler's Platform View factory to create and pass in a different configuration:

```
#if IOS || MACCATALYST
Microsoft.Maui.Handlers.WebViewHandler.PlatformViewFactory = (handler) =>
{
	var config = Microsoft.Maui.Platform.MauiWKWebView.CreateConfiguration();
	config.AllowsAirPlayForMediaPlayback = true;
	config.AllowsInlineMediaPlayback = true;
	config.AllowsPictureInPictureMediaPlayback = true;
	config.MediaTypesRequiringUserActionForPlayback = WebKit.WKAudiovisualMediaTypes.None;
	
	var wv = new Microsoft.Maui.Platform.MauiWKWebView(
		CoreGraphics.CGRect.Empty,
		handler as Microsoft.Maui.Handlers.WebViewHandler,
		config);

	return wv;
};
#endif
```

However for now, this sets the defaults to allow the inline media which is consistent with android/windows.

### TODO
- [x] Consider BlazorWebView default changes
- [x] Tests

### Issues Fixed
Fixes #4807

### CODE REVIEW

File: src/Core/src/Platform/iOS/MauiWKWebView.cs [hunk 1]
```diff
@@ -130,8 +130,20 @@ public void LoadUrl(string? url)
 		// It also has to be shared at the point you call init
 		public static WKWebViewConfiguration CreateConfiguration()
 		{
+			// By default, setting inline media playback to allowed, including autoplay
```
Comment (Redth) [inline src/Core/src/Platform/iOS/MauiWKWebView.cs:133]: @mkArtakMSFT @Eilon  @TanayParikh looking for BlazorWebView input... Given this explanation, would it make sense to setup the same defaults for Blazor's webview? Is this needed there?

Comment (TanayParikh) [reply src/Core/src/Platform/iOS/MauiWKWebView.cs:133]: This looks good to me 😄

**Review by TanayParikh — APPROVED** [review]
Blazor bits LGTM, can't comment on the rest :)

**Review by hartez — APPROVED** [review]

**Review by Eilon — APPROVED** [review]
The Blazor change looks good to me too. Blazor Hybrid users have asked for it!

**Review by rmarinho — CHANGES_REQUESTED** [review]
just iOS test failing 

image

**Review by rmarinho — APPROVED** [review]

**Pull request merged**